index.html

```diff
@@ -227,5 +235,29 @@ <h1>Skills</h1>
             </div>
         </div>
     </section>
+
+    <footer class="contact">
+        <div>
+            <h5>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h5>
+            <br>
+            <form action="https://formspree.io/f/xrgvqloo" method="POST">
+                <input type="text" name="tect" maxlength="30" placeholder="Full name" required>
+                <input type="email" name="email" placeholder="Email adress" required>
+                <textarea maxlength="500" name="message" placeholder="Enter text here" required></textarea>
+                <button type="submit" name="submit">Get in touch</button>
```

Comment:
You have done a great job so far @yegonkimutai 💪 but perhaps you didn't understand what the previous reviewer meant. I think the three color options are in triplets and so for instance, if your enabled button is green then the other states of the button should be in the other shades of green as provided and if your enabled state is blue then the other states of the button should be in the other shades of blue as provided 😉. Currently you have green on your enabled state and blue on your hover state : 
![image](https://user-images.githubusercontent.com/107795045/220453073-9ccd7c8b-28e9-4b6b-bc22-0b574739c307.png) 
![image](https://user-images.githubusercontent.com/107795045/220453173-4c5ef39f-b9bc-47b3-8ff4-1e42ac94f73e.png)